src/video/windows/SDL_windowsclipboard.c

```diff
@@ -69,7 +68,7 @@ static HANDLE WIN_ConvertBMPtoDIB(const void *bmp, size_t bmp_size)
 {
     HANDLE hMem = NULL;
 
-    if (bmp && bmp_size > sizeof(BITMAPFILEHEADER) && SDL_memcmp(bmp, bmp_magic, sizeof(bmp_magic)) == 0) {
+    if (bmp && bmp_size > sizeof(BITMAPFILEHEADER) && ((BITMAPFILEHEADER *)bmp)->bfType == BFT_BITMAP) {
```

Comment:
This assumes a little-endian machine, which I think is fine, but maybe better not to leave that footgun here? If you really want to do this, you should probably use the SDL endian swap functions to guarantee the value.

Reply:
Windows has always been little endian, even when they had played with ppc back in the day, so it should be OK?

Reply:
Good point. Okay, I'll merge as-is.